tests/run-make/link-framework/Makefile

```diff
@@ -1,23 +0,0 @@
-# only-apple
-#
-# Check that linking to a framework actually makes it to the linker.
-
-include ../tools.mk
-
-all:
-	$(RUSTC) dep-link-framework.rs
-	$(RUSTC) dep-link-weak-framework.rs
-
-	$(RUSTC) empty.rs
-	otool -L $(TMPDIR)/no-link | $(CGREP) -v CoreFoundation
```

Comment:
Discussion: this slightly baffles me as to how does this command *not* fail in the Makefile?

When I try

```
echo Foo | grep -v Foo
echo $?                   # <- exit status 1

exit 1 | grep -v Foo
echo $?                   # <- exit status 1
```

And AFAICT no-link does not exist. How did this ever work?

Reply:
EDIT: as 522 thankfully pointed out on discord, we actually need `run_fail` here, because the whole line's exit status is that of cgrep, not otool...

Reply:
I was going in circle in short. XD

Thanks so much!